CMakeLists.txt

```diff
@@ -215,3 +215,17 @@ if (test)
   add_dependencies(bdmclean testbdmclean)
 endif()
 
+#--------------------- build visualization projects ----------------------
+# build viz executable for small network test
+set(SMALL_NW_VIZ small-network-viz)
+add_executable(${SMALL_NW_VIZ} ${SOURCES} viz/small_network_visualization.cpp)
+target_include_directories(${SMALL_NW_VIZ} PUBLIC include)
+target_link_libraries(${SMALL_NW_VIZ} biodynamo)
+target_link_libraries(${SMALL_NW_VIZ} ${ROOT_LIBRARIES})
+
+# build viz executable for dividing cell test
+set(DIVIDING_CELL_VIZ dividing-cell-viz)
+add_executable(${DIVIDING_CELL_VIZ} ${SOURCES} viz/dividing-cell-viz.cpp)
```

Comment:
And `target_link_libraries(${SMALL_NW_VIZ}` ${ROOT_LIBRARIES})` neither as it is pulled in via the line above adding the biodynamo lib.